lib/javascript.d.ts

```diff
@@ -341,7 +346,7 @@ declare global {
 			 * this can be called synchronously and immediately returns the state.
 			 * Otherwise you need to provide a callback.
 			 */
-			getState: (callback?: GetStateCallback) => void | State;
+			getState: (callback?: GetStateCallback) => void | State | AbsentState;
```

Comment:
This is wrong. `getState` on a QueryResult that did not find any states returns null, not an absent state:

Short example: 
```js
log("QueryResult absent: " + JSON.stringify($("does.not.exist").getState()));
log("getState absent: " + JSON.stringify(getState("does.not.exist")));
```
yields
```
QueryResult absent: null
getState absent: {"val":null,"notExist":true}
```

Nonetheless, the signature should be broken into two overloads. One with required callback and void return type. And one with no callback and `State | null | undefined` return type.

Reply:
The AbsentState should instead be added to the "normal" getState signature (used to be in line 652)